PhysicsTools/NanoAOD/python/globals_cff.py

```diff
@@ -70,5 +71,17 @@
     ),
 )
 
+tauSpinnerTable = tauSpinnerTableProducer.clone(
+    src = 'prunedGenParticles',
+    name = 'TauSpinner',
+    theta = [0, 0.25, 0.5, -0.25, 0.375],
+    pdfSet = 'NNPDF31_nnlo_hessian_pdfas',
```

Comment:
How is the `pdfSet` used? Does it need to match the PDF used in the sample production?

Reply:
I think that PDFSet used by TauSpinner is relevant when TauSpinner is used to weight Z polarization, but not in the case of CP in the H->tau,tau decays. I will check it and confirm here.
After said this, I think that it will be good to match it with PDF used for sample production, but I don't think it is possible to check it automatically. Is it possible. Anyway, is it not NNPDF 3.1 NNLO which is used by default in current MC productions?

Reply:
I checked it again and I confirm that `PDFSet` and `cmsE` are important only for weighting Z/gamma* events. It is as polarization of Z/gamma* and then taus from its decays depends on production process where `PDFSet` and `cmsE` are important inputs. It is not the case for H->tau,tau.
As an "experimental" confirmation of this fact I ran the module with three different PDF sets and different collision energy, and I got identical results.
Because the parameters are irrelevant, I will remove them from list of configurable parameters, and set them to "sensible" defaults with an appropriate comment. When one will want to extend the module for Z/gamma*->tau,tau will clearly know what should be done.